test/tests/selftest/pbs_default_timeout.py

```diff
@@ -36,32 +36,57 @@
 # trademark licensing policies.
 
 from tests.selftest import *
+from ptl.utils.plugins.ptl_test_runner import TimeOut
 
 
 class TestDefaultTimeout(TestSelf):
     """
     Test suite to verify increase default testcase timeout working properly.
     """
 
+    @staticmethod
+    def common_sleep_function(sleep_val):
+        time.sleep(sleep_val)
+
     def test_default_timeout(self):
         """
         Verify that test not timedout after 180 sec.
         i.e. after previous default timeout value
         """
-        time.sleep(300)
+        try:
+            self.common_sleep_function(MINIMUM_TESTCASE_TIMEOUT/2)
+        except TimeOut:
+            mssg = "Test Timeout before complete default-testcase-timeout"
+            raise TimeOut(mssg)
+
+    def test_timeout_greater_default_value(self):
+        """
+        Verify that test timedout after 600 sec.
+        """
+        mssg = 'Timed out after 600 second'
```

Comment:
shouldn't it be 600 seconds (not second) ?

Reply:
This message is from timeout_handler function.

def timeout_handler(signum, frame):
    raise TimeOut('Timed out after %s second' % timeout)

Reply:
i think it's safe to change the string to 'seconds', changes of the timeout period being 1 second is pretty small

Reply:
I'm okay with you leaving this as is as the message is generated by PTL, other tests might also be searching for the same string, so we can deal with that separately.